### DISCUSSION TITLE
Using docker for local development

### DISCUSSION THREAD

**Discussion by alexjamesbrown** [medusajs/medusa#4984] [Q&A]
I'm trying to get a local environment setup to build a medusajs store

I've used this https://github.com/medusajs/docker-medusa/
But I'm a bit confused.

I'm guessing `admin` isn't required as a standalone, as it's bundled into `backend` ?

I've added https://github.com/medusajs/nextjs-starter-medusa to the `storefront` dir

When I do `docker compose up --build` it starts, and if I go to http://localhost:9000/store/products it returns the products in json.
But I can't get to admin.
If I go to http://localhost:9000/app it just says 
> Admin is running in development mode. See the terminal for the URL.

Bit lost from here...

My attempt so far is at:
https://github.com/alexjamesbrown/medusa-test

**Comment by pevey** [comment]
In development mode, the admin runs at http://localhost:7001

Is there a way with Docker to see a stream of the console logs from the medusa backend?  That is where that URL would have been shown.  For development purposes, lots of other useful debugging messages are shown there in dev mode.  It will be very helpful to you if you can find a way to see the logs. 

Medusa also has the ability to log those messages to file a now, since v 1.15, I think.  Set the environment variable LOG_FILE to whatever you want the file to be. (https://github.com/medusajs/medusa/pull/4696)
